.changelog/unreleased/breaking-changes/470-standalone-commitment-computation.md

```diff
@@ -0,0 +1,3 @@
+- Export standalone commitment computations for `PacketCommitment` and
```

Comment:
We would change this to something more along the lines of "made packet/ack commitment methods private"

Reply:
[587fa8b](https://github.com/cosmos/ibc-rs/pull/492/commits/587fa8bab4b78dd48f77a3af70f1954943164e44)